### DISCUSSION TITLE
How is the progress of multilingual support?

### DISCUSSION THREAD

**Discussion by uoneway** [jbesomi/texthero#203] [Q&A]
Hello, I'm Hangil.
I got to know this wonderful project a few days ago,
and can see some issues related with multilingual support written a few months ago
https://github.com/jbesomi/texthero/issues/84
https://github.com/jbesomi/texthero/pull/128

How is the progress of multilingual support?
If it is in progress, I want to contribute for Korean.

Please let me know :)

**Comment by jbesomi** [comment]
Hi @uoneway,

Thank you for your message!

We are working on multilingual support, we haven't made much progress yet, the goal here is to first find a solution that's as general as possible to avoid over-engineering and to keep Texthero simple.

Can you kindly answer these questions? Thanks!

 - In which manner an NLP Korean pipeline differ from an English NLP pipeline?
 - How did you know about Texthero?
 - What functions and modules of Texthero you used until now? Could you use it with Korean text? If not, what should be changed?
 - How much effort (in hour) do you think you can put in helping with multilingual support?
